tests/foreman/rhai/test_rhai.py

```diff
@@ -421,6 +421,30 @@ def test_positive_inventory_export_csv():
     pass
 
 
+@pytest.mark.stubbed
+def test_positive_inventory_generate_upload_cli():
+    """Tests Insights inventory generation and upload via foreman-rake commands.
+
+    :id: f2da9506-97d4-4d1c-b373-9f71a52b8ab8
+
+    :Steps:
+
+        0. Create a VM and register to insights within org having manifest.
+
+        1. Run 'foreman-rake rh_cloud_inventory:report:generate_upload' on the Satellite.
+
+        2. Run 'rh_cloud_inventory:report:generate' and 'rh_cloud_inventory:report:upload'.
```

Comment:
`target=/path/to/directory/you/have/permissions/to organization_id=1 foreman-rake rh_cloud_inventory:report:generate` we should include `target` and `organization_id` as well.

Reply:
Updated.